db/schema.rb

```diff
@@ -141,8 +141,9 @@
   create_table "councillor_contributions", force: :cascade do |t|
     t.integer  "contributor_id", limit: 4
     t.integer  "authority_id",   limit: 4
-    t.datetime "created_at",               null: false
-    t.datetime "updated_at",               null: false
+    t.datetime "created_at",                 null: false
+    t.datetime "updated_at",                 null: false
+    t.string   "source_url",     limit: 255
```

Comment:
If we change this attribute name to just be 'source' we should change it here too.

Also, because people might want to tell a bit of a story here, and we welcome that, we should use the column data type 'text' rather than 'string'. Text is generally for larger blocks and defaults to having a larger limit.